<launch>
    <arg name="wheel_diameter" default="0.165"/>
    <arg name="wheel_separation" default="0.385"/>
    <arg name="publish_frequency" default="10"/>
    <arg name="publish_tranform" default="false"/>

    <node pkg="sr_kinematic" type="sr_kinematic" name="sr_kinematic" output="screen">
        <rosparam command="load" file="$(find sr_kinematic)/config/differential_2d_robot.yaml"/>
        <param name="wheel_diameter" value="$(arg wheel_diameter)"/>
        <param name="wheel_separation" value="$(arg wheel_separation)"/>
        <param name="publish_frequency" value="$(arg publish_frequency)"/>
        <param name="publish_tranform" value="$(arg publish_tranform)"/>
    </node>

    <!-- tranform between laser_frame and base_link
    <node pkg="tf2_ros" type="static_transform_publisher" name="base_link_to_laser_frame_broadcaster" args="0.175 0.0 0.115 0.0 0.0 0.0 base_link laser_frame" output="screen"/>

    <node pkg="tf2_ros" type="static_transform_publisher" name="base_link_to_imu_link_broadcaster" args="0.0 0.0 0.0 0.0 0.0 0.0 base_link imu_link" output="screen"/> -->
</launch>
